### DISCUSSION TITLE
Implementation of a Baseline Feature for Error and Warning Suppression in BiomeJs

### DISCUSSION THREAD

**Discussion by urbicon** [biomejs/biome#1064] [RFC]
Hello BiomeJs Community,

I am proposing a new feature to introduce the concept of a 'baseline' for projects within BiomeJs. This would allow the tool to record the current state of a project's errors and warnings and suppress their repeated output in future scans, while still summarizing them in reports. This approach is similar to practices in PHP environments and would be beneficial for maintaining legacy codebases, focusing on newly introduced issues.

I have checked the playground and existing issues and have not found a similar feature. 

Thank you for considering this proposal, and I look forward to the community's feedback and suggestions.

Cheers,
Felix

**Comment by Conaclos** [comment]
I think this has already been brought up in [another discussion] (https://github.com/biomejs/biome/discussions/339). I really like this idea.

If someone is interested to implement this, I encourage him/her to first open an RFC discussion detailing the feature (behaviors, CLI options, ...).
